sql/hive/pom.xml

```diff
@@ -168,6 +128,18 @@
       <groupId>org.datanucleus</groupId>
       <artifactId>datanucleus-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.datanucleus</groupId>
```

Comment:
Why handle `datanucleus` separately?

Reply:
Does spark use datanucleus directly?

Reply:
No, but I think it's a bit risky to relocate datanucleus. If you look at [HiveUtils](https://github.com/apache/spark/blob/master/sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveUtils.scala#L518) there are places where class name from datanucleus is directly referenced. After relocation it may not function properly.

Reply:
Got it. Thanks for the explanation.